_posts/2019-11-13-Extend_Your_Data_Center_to_the_Hybrid_Edge-Red_Hat_Summit.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Extend Your Data Center to the Hybrid Edge - Red Hat Summit, May 2019"
+date: 2019-11-13T10:37:00+02:00
```

Comment:
Maybe we should be making the dates of these posts more closely match the original dates of the events?

Reply:
In future events/video blogposts yes, with no doubt. Do you think is necessary for those ones?

Reply:
And by "those ones" I mean the ones we are creating now that refer to events in the past. Another thing is that if we change the date for an older one the blogpost would be list in that order and will lose the natural visibility of a new post being in the top. Whatever option you prefer just tell me and I'll make the changes to publish it asap.

Reply:
@dhellmann what are we doing with this? if you still want to change the date we can write 1st of April as the event happened in May. Otherwise, we can publish as it is, any thoughts?